hera_qm/data/a_priori_flags_integrations.yaml

```diff
@@ -0,0 +1,40 @@
+# A PRIORI FLAGS
+
+# The following flags can be loaded by functions in hera_qm.metrics_io, including:
+# read_a_priori_chan_flags(), read_a_priori_int_flags(), and read_a_priori_ant_flags().
+# Variable names are hard-coded and cannot be changed without unexpected behavior.
+
+# ---------------
+# FREQUENCY FLAGS
+# ---------------
+
+# freq_flags are an array of length-2 arrays of floats in units of Hz (freq ranges, inclusive)
+freq_flags: [[0, 110e6],
+             [150e6, 155e6],
+             [190e6, 200e6]]
+
+# channel_flags are an array of ints or length-2 arrays of ints (channel ranges, inclusive)
+channel_flags: [0, 1, [10, 20], 60]
+
+
+# ---------------
+# TIME FLAGS
+# ---------------
+
+# # JD_flags are an array of length-2 arrays of floats in units of days (time ranges, inclusive)
+JD_flags: []
```

Comment:
FYI, instead of leaving them empty you can also choose not to have these fields. That's your call.

Reply:
I'll just leave them empty.

Reply:
leaving it empty.